core/app/components/pubs/PubEditor/PubEditorClient.tsx

```diff
@@ -1,348 +1,348 @@
 "use client";
```

Comment:
this isn't a very useful diff—it would be more useful to diff this against `ExternalFormWrapper`! see diagram in PR description for more info